.github/workflows/ci.yml

```diff
@@ -342,16 +342,16 @@ jobs:
       env:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
     - name: Compress logs
-      if: steps.build.outcome == 'success'
+      if: ${{ always() }}
```

Comment:
@robertomonteromiguel 

if build step is not a success, there is no log to push? did I missed something  ?

Reply:
What was happening is that if any tests failed, the logs were not being uploaded.

Reply:
strange, do you know why. From what i understand, the condition is only on build step, no ?